### PR TITLE
data: update latest 4.17.1.0 release

### PR DESCRIPTION
**Pull request by shwstppr**
Reverts commit https://github.com/apache/cloudstack-www/commit/61c3d06c787042ea82cd761114128ce37bd275dc.

### CODE REVIEW

File: data/cloudstack.yml [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
   main_islts: yes
   main_docs_url: https://docs.cloudstack.apache.org/en/4.17.0.0/
   branch: 4.17
-  lts: 4.17.0.1
+  lts: 4.17.1.0
   lts_docs_url: https://docs.cloudstack.apache.org/en/latest/installguide/index.html
   ltsbranch: 4.17
   archives:
     - '4.17':
-       - 4.17.0.1
```
Comment (shwstppr) [inline data/cloudstack.yml:11]: @rohityadavcloud should we keep 4.17.0.1 release here?

File: content/archives.html [hunk 2]
```diff
@@ -125,13 +125,11 @@ <h2 id="indicators">Archived Releases</h2>
 <ul>
   <li>4.17
     <ul>
-      <li><a href="https://archive.apache.org/dist/cloudstack/releases/4.17.0.1/apache-cloudstack-4.17.0.1-src.tar.bz2">apache-cloudstack-4.17.0.1.src.tar.bz2</a> [ <a href="https://archive.apache.org/dist/cloudstack/releases/4.17.0.1/apache-cloudstack-4.17.0.1-src.tar.bz2.asc">PGP</a> ][ <a href="https://archive.apache.org/dist/cloudstack/releases/4.17.0.1/apache-cloudstack-4.17.0.1-src.tar.bz2.sha512">SHA512</a> ]</li>
```
Comment (rohityadavcloud) [inline content/archives.html:128]: re-add those

File: content/archives.html [hunk 3]
```diff
@@ -125,13 +125,11 @@ <h2 id="indicators">Archived Releases</h2>
...
       <li><a href="https://archive.apache.org/dist/cloudstack/releases/4.17.0.0/apache-cloudstack-4.17.0.0-src.tar.bz2">apache-cloudstack-4.17.0.0.src.tar.bz2</a> [ <a href="https://archive.apache.org/dist/cloudstack/releases/4.17.0.0/apache-cloudstack-4.17.0.0-src.tar.bz2.asc">PGP</a> ][ <a href="https://archive.apache.org/dist/cloudstack/releases/4.17.0.0/apache-cloudstack-4.17.0.0-src.tar.bz2.sha512">SHA512</a> ]</li>
     </ul>
   </li>
   <li>4.16
     <ul>
-      <li><a href="https://archive.apache.org/dist/cloudstack/releases/4.16.1.1/apache-cloudstack-4.16.1.1-src.tar.bz2">apache-cloudstack-4.16.1.1.src.tar.bz2</a> [ <a href="https://archive.apache.org/dist/cloudstack/releases/4.16.1.1/apache-cloudstack-4.16.1.1-src.tar.bz2.asc">PGP</a> ][ <a href="https://archive.apache.org/dist/cloudstack/releases/4.16.1.1/apache-cloudstack-4.16.1.1-src.tar.bz2.sha512">SHA512</a> ]</li>
```
Comment (rohityadavcloud) [inline content/archives.html:134]: re-add those

File: content/downloads.html [hunk 4]
```diff
@@ -130,63 +130,63 @@ <h2>Latest CloudStack Releases</h2>
 <p>The CloudStack community maintains two types of releases; the main releases and the LTS (Long Term Support) releases. The LTS releases receive bug and security fixes for a period of 18 months after the main release (for more details <a href="https://cwiki.apache.org/confluence/display/CLOUDSTACK/LTS"><i>click here</i></a>). The main releases receive only critical bug fixes for a short period. The general expectation is that the users of the main version will upgrade to a new version in order to receive fixes.</p>
 
 <h3>Source Releases</h3>
-<p>Apache CloudStack's most recent release is 4.17.0.1.
+<p>Apache CloudStack's most recent release is 4.17.0.0.
```
Comment (rohityadavcloud) [inline content/downloads.html:133]: fix to say 4.17.1.0 here and all below?

File: data/cloudstack.yml (hunk 1 above)
Comment (rohityadavcloud) [reply data/cloudstack.yml:11]: Yes, we should add both

File: data/cloudstack.yml [hunk 5]
```diff
@@ -1,17 +1,17 @@
 versions:
-  main: 4.17.0.1
+  main: 4.17.1.0
   main_islts: yes
   main_docs_url: https://docs.cloudstack.apache.org/en/4.17.0.0/
```
Comment (rohityadavcloud) [inline data/cloudstack.yml:4]: do url to 4.17.1.0 pls

File: data/cloudstack.yml (hunk 1 above)
Comment (shwstppr) [reply data/cloudstack.yml:11]: Fixed

File: content/archives.html (hunk 2 above)
Comment (shwstppr) [reply content/archives.html:128]: Fixed

File: content/downloads.html (hunk 4 above)
Comment (shwstppr) [reply content/downloads.html]: Fixed

File: data/cloudstack.yml (hunk 5 above)
Comment (shwstppr) [reply data/cloudstack.yml]: Fixed

**Review by rohityadavcloud — APPROVED** [review]

**Comment by rohityadavcloud** [comment]
Will merge this and test, we won't make it live until it passes tests and all pkgs are uploaded/fixed as well as the docs are live.

**Pull request merged**